src/citra_qt/configuration/configure_general.cpp

```diff
@@ -56,6 +59,15 @@ ConfigureGeneral::ConfigureGeneral(QWidget* parent)
                     .rightJustified(tr("unthrottled").size()));
         }
     });
+
+    connect(ui->change_screenshot_dir, &QToolButton::clicked, this, [this] {
+        const QString dir_path = QFileDialog::getExistingDirectory(
+            this, tr("Select Screenshot Directory"), UISettings::values.screenshot_path,
```

Comment:
```suggestion
            this, tr("Select Screenshot Directory"), ui->screenshot_dir_path->text(),
```
In case this was already changed after loading the configuration.